cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -54,6 +54,8 @@ Replacement syntax for deprecated and removed features are also indicated.
 | `ON DEFAULT DATABASE` for privilege commands  | Syntax        | Deprecated    | Replaced by `ON HOME DATABASE`
 | `ON DEFAULT GRAPH` for privilege commands     | Syntax        | Deprecated    | Replaced by `ON HOME GRAPH`
 | `CREATE INDEX [name] FOR ()-[r:TYPE]-() ON (r.prop)` | Functionality | Added | Allows creating indexes on relationships.
+| `RENAME ROLE` | Syntax | Added | New Cypher command for changing the name of a role
+| `RENAME USER` | Syntax | Added | New Cypher command for changing the name of a user
```

Comment:
I wonder if these should be `Functionality` instead of `Syntax`

Reply:
Good catch!

Reply:
I wonder if we should try to sort that list a bit in the end of 4.3 cycle, either by schema/administration/Cypher or added/updated/deprecated, now it starting to look a bit messy

Reply:
it is mostly ordered by added to the bottom :P but yes it might be nice to order it a bit better